turbine/src/cluster_nodes.rs

```diff
@@ -362,7 +362,15 @@ impl<T: 'static> ClusterNodesCache<T> {
         working_bank: &Bank,
         cluster_info: &ClusterInfo,
     ) -> Arc<ClusterNodes<T>> {
-        let epoch = root_bank.get_leader_schedule_epoch(shred_slot);
+        let epoch = if check_feature_activation(
+            &feature_set::revise_turbine_epoch_stakes::id(),
+            shred_slot,
+            root_bank,
+        ) {
+            root_bank.epoch_schedule().get_epoch(shred_slot)
```

Comment:
do we need to also use `epoch_schedule()` below in the fallback case where epoch-stakes aren't available from either `root_bank` or `working_bank` for the shred slot and we use epoch-stakes from the root_banks's slot instead?

Reply:
done.